### ISSUE TITLE
Support enabling other Alloy components which mount files

### ISSUE THREAD

**Issue by iainlane** [grafana/k8s-monitoring-helm#726]
I'm set up with k8s-monitoring in Grafana Cloud and it's working well, great job team 👐. Next I'd like to use other components from the Alloy deployed by this chart, so I don't have to manage a separate instance.

Is that something you support / recommend? If not, stop here and close this issue. 😁

Ok!

For that I was thinking I'd use this config settting

https://github.com/grafana/k8s-monitoring-helm/blob/388adca46e3fa8ff5ebcd90888549349ff07673a/charts/k8s-monitoring/values.yaml#L1974-L1979

The component I'm interested in right now is [`prometheus.exporter.snmp`](https://grafana.com/docs/alloy/latest/reference/components/prometheus/prometheus.exporter.snmp/). As you can see on those docs, I need to supply a path to a config file. I can't find a way to do this in the chart as it stands now.

Would you consider providing a way for me to give either file contents / path pairs inline, which get mounted into the pod, or perhaps references to config maps which get volume mounted?

**Comment by iainlane** [comment]
Yes totally. They seem to essentially be the same case but with different paths.

**Comment by dmajano** [comment]
I'm facing the same situation. I need to load a yaml file for blackbox exporter for monitor websites but I don't find How i can mount a config map in the helm chart
